react/Checkbox/Checkbox.js

```diff
@@ -28,15 +28,17 @@ export default class Checkbox extends Component {
       onChange: PropTypes.func,
       checked: PropTypes.bool.isRequired
     }),
-    type: PropTypes.oneOf([STANDARD, BUTTON])
+    type: PropTypes.oneOf([STANDARD, BUTTON]),
+    isMultiline: PropTypes.bool
```

Comment:
Can we drop the `is` prefix from this? We don't tend to prefix props this way (even though I think it makes a lot of sense)

In fact, maybe `wrap` is a more appropriate name, since it's not *necessarily* going to render to lines?